diff --git a/src/xoj-preview-extractor/CMakeLists.txt b/src/xoj-preview-extractor/CMakeLists.txt
index cca14e1a..a95aad77 100644
--- a/src/xoj-preview-extractor/CMakeLists.txt
+++ b/src/xoj-preview-extractor/CMakeLists.txt
@@ -39,9 +39,9 @@ target_include_directories (xournalpp-thumbnailer PRIVATE ${librsvg_INCLUDE_DIRS
 
 set (THUMBNAILER_BIN "xournalpp-thumbnailer")
 
-add_custom_command (TARGET xournalpp-thumbnailer POST_BUILD
-  COMMAND ${CMAKE_STRIP} ${THUMBNAILER_BIN}
-)
+#add_custom_command (TARGET xournalpp-thumbnailer POST_BUILD
+#  COMMAND ${CMAKE_STRIP} ${THUMBNAILER_BIN}
+#)
 
 ## Install ##
 
